src/shared/form_fields/user_my_details.js

```diff
@@ -0,0 +1,56 @@
+
+export const contact_fields =
```

Comment:
I think we should have moved `src/admin/form_fields/member.js` here and built on that